backend/Moenda/MD/MDRules.js

```diff
@@ -0,0 +1,122 @@
+const util = require("../util");
```

Comment:
Why are there the same files of moenda? Isn't better to communicate with the moenda app instead of replicate each file? How do you plan keep them sync?

Reply:
My suggestion is download the moenda app and use the modules exported by the app